main.rb

```diff
@@ -0,0 +1,94 @@
+module Enumerable
+  def my_each
+    index = 0
+
+    while index < size
+      yield self[index]
+      index += 1
+    end
+  end
```

Comment:
- [ ] should have working `my_each` identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))


Yours is not:
- Returning an enumerator when no block is given.